package.json

```diff
@@ -53,13 +56,17 @@
     "gulp-replace": "1.0.0",
     "gulp-sourcemaps": "2.6.5",
     "gulp-terser": "1.4.0",
+    "highcharts": "7.2.1",
```

Comment:
@alexhoffman617 This appears to be the cause of the Snyk error. Looks like a bad rebase or merge, since during the course of this PR we updated to highcharts 7.2.2, which is on the line below.

```suggestion
```

Reply:
@Scotchester  got rid of the duplication, there still seems to be an error but it is no longer blocking the merge